annotations/mozilla_vpn/metrics/sample.addon_message_state_changed/README.md

```diff
@@ -0,0 +1 @@
+<img width="633" alt="Screenshot 2023-04-12 at 10 22 54 PM" src="https://user-images.githubusercontent.com/49305857/231631119-e5ac5059-5710-4278-bf9f-38fa750e92f5.png">
```

Comment:
@badboy @Iinh do you think we should store images differently here?

Reply:
The image was probably dropped into the editor on github.com editing that file, which is why it's uploaded and has width and alt.
I don't think there's an easy way to add the file + README changes in a single edit in the github UI, but I can do that on behalf of Yeonjoo

Reply:
@badboy thanks!